config/locales/en.yml

```diff
@@ -157,9 +157,6 @@ en:
               <<: *report_dates
               before_start_date: "can't be after Date to"
 
-  page_title:
```

Comment:
Does this need to be removed?

Reply:
Yes all the other page titles are in `page_titles.yml` so I think the left over `marking_response_as_sent` was a mistake.